app/(drawer)/(tabs)/library/music/playlists/[playlistId]/edit.tsx

```diff
@@ -48,61 +102,90 @@ export default function PlaylistsPage() {
+          const index = getIndex();
+          const { id, title, artist, artworkUrl } = item;
           const isLastRow = index === tracks.length - 1;
           const marginBottom = isLastRow ? height + 16 : 16;
 
           return (
-            <View
-              style={{
-                flexDirection: "row",
-                marginBottom,
-                alignItems: "center",
-                gap: 4,
-              }}
-            >
-              {/* TODO - swap placeholder with artwork of first track */}
-              <LogoIcon fill="white" width={60} height={60} />
-              {/* <SquareArtwork size={60} url={artworkUrl} /> */}
-              <View style={{ marginLeft: 10, flex: 1 }}>
-                <Text
-                  style={{
-                    fontSize: 18,
-                  }}
-                  numberOfLines={3}
-                  bold
-                >
-                  {title}
-                </Text>
-                <Text
-                  style={{
-                    fontSize: 18,
-                  }}
-                  numberOfLines={3}
-                  bold
+            <>
+              <ScaleDecorator>
+                <TouchableOpacity
+                  activeOpacity={1}
+                  onPressIn={drag}
+                  disabled={isActive}
                 >
-                  {artist}
-                </Text>
-              </View>
-            </View>
+                  <View
+                    style={{
+                      flexDirection: "row",
+                      marginBottom,
+                      alignItems: "center",
+                      gap: 4,
+                    }}
+                  >
+                    <Pressable
+                      style={{
+                        width: 40,
+                        height: 40,
+                        flexDirection: "row",
+                        alignItems: "center",
+                        justifyContent: "center",
+                      }}
+                      onPress={() => handleDelete(index)}
+                    >
+                      <MaterialCommunityIcons
+                        name={"delete"}
+                        size={30}
+                        color={"red"}
+                      />
+                    </Pressable>
+                    <SquareArtwork size={60} url={artworkUrl} />
+                    <View style={{ marginLeft: 10, flex: 1 }}>
+                      <Text
+                        style={{
+                          fontSize: 18,
+                        }}
+                        numberOfLines={3}
+                        bold
+                      >
+                        {title}
+                      </Text>
+                      <Text
+                        style={{
+                          fontSize: 18,
+                        }}
+                        numberOfLines={3}
+                        bold
+                      >
```

Comment:
To match other views:
```suggestion
                      <Text
                        numberOfLines={1}
                      >
```